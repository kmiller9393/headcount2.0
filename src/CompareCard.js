import React from 'react';
import './Card.css';
import PropTypes from 'prop-types';

const CompareCard = ({
  DistrictOne,
  DistrictOneData,
  comparison,
  DistrictTwo,
  DistrictTwoData
}) => {
  return (
    <div className="Card Chosen Compared">
      <h2 className="top-district">
        {DistrictOne}: {DistrictOneData}
      </h2>
      <h2 className="comparison">compared: {comparison}</h2>
      <h2 className="bottom-district">
        {DistrictTwo}: {DistrictTwoData}
      </h2>
    </div>
  );
};

CompareCard.propTypes = {
  DistrictOne: PropTypes.string,
  DistrictOneData: PropTypes.number,
  comparison: PropTypes.number,
  DistrictTwo: PropTypes.string,
  DistrictTwoData: PropTypes.number
};

export default CompareCard;
