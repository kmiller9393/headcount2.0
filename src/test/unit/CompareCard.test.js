import React from 'react';
import CompareCard from '../../CompareCard';
import { shallow, mount } from 'enzyme';
