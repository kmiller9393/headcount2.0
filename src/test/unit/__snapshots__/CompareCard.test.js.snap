// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompareCard matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CompareCard
    DistrictOne="COLORADO"
    DistrictOneData={0.53}
    DistrictTwo="ADAMS COUNTY 14"
    DistrictTwoData={0.709}
    comparison={0.748}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          COLORADO
          : 
          0.53
</h2>,
        <h2>
          0.748
</h2>,
        <h2>
          ADAMS COUNTY 14
          : 
          0.709
</h2>,
      ],
      "className": "Card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "COLORADO",
            ": ",
            0.53,
          ],
        },
        "ref": null,
        "rendered": Array [
          "COLORADO",
          ": ",
          0.53,
        ],
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": 0.748,
        },
        "ref": null,
        "rendered": 0.748,
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "ADAMS COUNTY 14",
            ": ",
            0.709,
          ],
        },
        "ref": null,
        "rendered": Array [
          "ADAMS COUNTY 14",
          ": ",
          0.709,
        ],
        "type": "h2",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            COLORADO
            : 
            0.53
</h2>,
          <h2>
            0.748
</h2>,
          <h2>
            ADAMS COUNTY 14
            : 
            0.709
</h2>,
        ],
        "className": "Card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "COLORADO",
              ": ",
              0.53,
            ],
          },
          "ref": null,
          "rendered": Array [
            "COLORADO",
            ": ",
            0.53,
          ],
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": 0.748,
          },
          "ref": null,
          "rendered": 0.748,
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "ADAMS COUNTY 14",
              ": ",
              0.709,
            ],
          },
          "ref": null,
          "rendered": Array [
            "ADAMS COUNTY 14",
            ": ",
            0.709,
          ],
          "type": "h2",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
